### ISSUE TITLE
Ground Truth path

### ISSUE THREAD

**Issue by 32nguyen** [TristaZeng/ZS-DeconvNet#28]
Hi,

I wonder why the Ground Truth was involved in training process, for example in Train_ZSDeconvNet_2D.py, 

<img width="431" height="172" alt="Image" src="https://github.com/user-attachments/assets/72692adf-5d91-45df-9926-b865fc65a0f2" />

Are input and gt_g a pair of LR-HR images? So is this supervised method? 

From what I understand is that, as the unsupervised, the images used in the loss function should be a denoised image and a final high resolution image generated during training process. It seems to me that they are pre-prepared in "train_gt_path = data_dir+folder+'/gt' ". Would you please help me with this?

Thanks.
